ramls/inventory-hierarchy/inventory-instance-records.json

```diff
@@ -0,0 +1,813 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "description": "Inventory records view response schema",
+  "properties": {
+    "instanceId": {
+      "description": "Inventory instance identifier",
+      "type": "string",
+      "$ref": "../uuid.json"
+    },
+    "source": {
+      "description": "Source of metadata and format of the underlying record to the instance record",
+      "type": "string"
+    },
+    "holdings": {
+      "type": "array",
+      "description": "Holdings record fields",
+      "items": {
```

Comment:
Is this representation of holdings actively different to the main holdings definition?

Reply:
I think current represetnation of holdings contains combined parts of original ones. The same for items too. Because not all parts needed and some of them needs to be presented in extended forms, I mean "names" instead of id references to other wntities, for example.